src/officehours_api/backends/zoom.py

```diff
@@ -170,6 +179,9 @@ def _create_meeting(cls, user: User) -> ZoomMeeting:
                 },
             },
         )
+        if resp.status_code == 401:
```

Comment:
I think we're only likely to see `401` here if the token has been invalidated, but we could also use the client range if we were concerned. It's working like this for now.